docs/software/sciapps/vasp.md

```diff
@@ -1,5 +1,368 @@
 [](){#ref-uenv-vasp}
 # VASP
 
-!!! todo
-    port over docs
+The Vienna Ab initio Simulation Package ([VASP]) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles.
+
+VASP computes an approximate solution to the many-body Schrödinger equation, either within density functional theory (DFT), solving the Kohn-Sham equations, or within the Hartree-Fock (HF) approximation, solving the Roothaan equations.
+Hybrid functionals that mix the Hartree-Fock approach with density functional theory are implemented as well.
+Furthermore, Green's functions methods (GW quasiparticles, and ACFDT-RPA) and many-body perturbation theory (2nd-order Møller-Plesset) are available in VASP.
+
+In VASP, central quantities, like the one-electron orbitals, the electronic charge density, and the local potential are expressed in plane wave basis sets.
+The interactions between the electrons and ions are described using norm-conserving or ultrasoft pseudopotentials, or the projector-augmented-wave method.
+To determine the electronic groundstate, VASP makes use of efficient iterative matrix diagonalisation techniques, like the residual minimisation method with direct inversion of the iterative subspace (RMM-DIIS) or blocked Davidson algorithms.
+These are coupled to highly efficient Broyden and Pulay density mixing schemes to speed up the self-consistency cycle.
+
+
+!!! note "Licensing Terms and Conditions"
+    Users are kindly asked to obtain their own license. CSCS cannot provide free access to the code and needs to inform VASP Software GmbH with an updated list of users.
```

Comment:
```suggestion
    Access to VASP is restricted to users who have purchased a license from VASP Software GmbH.
    CSCS cannot provide free access to the code and needs to inform VASP Software GmbH with an updated list of users.
    Once you have a license, submit a request on the [CSCS service desk](https://jira.cscs.ch/plugins/servlet/desk) (with a copy of your license) to be added to the `vasp6` unix group, which will grant access to the `vasp` uenv.
    Please refer to the VASP web site for more information about licensing.
```

Reply:
Then remove the two sentences below that?

"users are kindly asked" is indirect and a bit unctuous.